src/org/loklak/data/ElasticsearchClient.java

```diff
@@ -988,4 +990,65 @@ public TermsBuilder getClassifierAggregation(String classifierName) {
                 AggregationBuilders.sum("sum_probability").field(probabilityField)
             );
     }
+
+
+    public HashMap<String, HashMap<String, Double>> classifierScore(String index, String classifierName, ArrayList<String> classes, String startDate, String endDate) {
```

Comment:
I have a doubt, does this method aggregate data acc. `to date` and acc. `to class` from query and then obtain some scores. What does class means here `sweat_smile`